content/06.know-your-problem.md

```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
 
 In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+One tool in the practicioner's toolkit for dealing with datasets that are not thoroughly understood is data simulation.
+Simulating data under reasonable assumptions and assessing the extent to which the model is capable of capturing or mitigating issues with certain features of the data is a valuable technique for identifying model architectures and hyperparameters that are reasonable under those assumptions.

```

Comment:
Just a small mod, but also here, changing to 

"the model is capable of capturing or mitigating issues with certain features"

 to 

 "the model can capture or mitigate issues with certain features"

could make it a tad more readable.